src/views/Timeline.vue

```diff
@@ -464,7 +472,7 @@ export default {
       if (inf.length > 0) {
         topLabel = inf[0].label
       }
-      let icon = 'mdi-crosshairs-question'
+      let icon = ''
```

Comment:
@vickywane several comments here.

1. I see you placed the fall detection icon rendering logic in the vue markup instead of reusing and generalizing this `eventIcon` function that specifically picks the correct icon based on the event type. I see that you did it because the fall icon requires use of a third party library (iconify) instead of the standard MDI set. However the code now looks disorganized and messy.
2. Let's create an `<event-icon />` component that encapsulates the ugly details of rendering the correct icon markup. Move `eventIcon()` and the fall detection icon markup out of `timeline.vue` and replace with a simple and elegant one line `<event-icon :event-type="$sample.args.inference_meta.display" />` vue component reference.
3. You removed the use of 'mdi-crosshairs-question'. Is the intention to not show any event icon at all if the event type is uknown?